examples/10. Data Structures.md

```diff
@@ -1,75 +1,401 @@
 FORMAT: 1A
+HOST: https://polls.apiblueprint.org/
 
-# Data Structures API
-Following [Advanced Attributes](09.%20Advanced%20Attributes.md), this example
-demonstrates defining arbitrary data structure to be reused by various
-attribute descriptions.
+# Maintenance
 
-Since a portion of the `Coupon` data structure is shared between the `Coupon`
-definition itself and the `Create a Coupon` action, it was separated into a
-`Coupon Base` data structure in the `Data Structures` API Blueprint Section.
-Doing so enables us to reuse it as a base-type of other attribute definitions.
+ตัวอย่างออกแบบ API สำหรับงานแจ้งซ่อม
```

Comment:
I'm just getting non renderable chars here, is it my browser (up to date ff on linux) that just broke or is this some non utf8 stuff?

Reply:
I can read it, but not sure what is it.